src/mlj_model_interface.jl

```diff
@@ -43,7 +49,26 @@ end
 const ERR_BUILDER =
     "Builder does not appear to build an architecture compatible with supplied data. "
 
-true_rng(model) = model.rng isa Integer ? MersenneTwister(model.rng) : model.rng
+true_rng(model) = model.rng isa Integer ? Random.Xoshiro(model.rng) : model.rng
+
+# Models implement L1/L2 regularization by chaining the chosen optimiser with weight/sign
+# decay.  Note that the weight/sign decay must be scaled down by the number of batches to
+# ensure penalization over an epoch does not scale with the choice of batch size; see
+# https://github.com/FluxML/MLJFlux.jl/issues/213.
+
+function regularized_optimiser(model, nbatches)
```

Comment:
So if I read this correctly, it is not possible to specify either L1 or L2? The regularized optimiser will always apply both?